rasa_core/evaluate.py

```diff
@@ -485,18 +556,37 @@ def plot_story_evaluation(test_y, predictions,
     logging.basicConfig(level=cmdline_args.loglevel)
     _endpoints = AvailableEndpoints.read_endpoints(cmdline_args.endpoints)
 
-    _interpreter = NaturalLanguageInterpreter.create(cmdline_args.nlu,
-                                                     _endpoints.nlu)
-
-    _agent = Agent.load(cmdline_args.core,
-                        interpreter=_interpreter)
-
-    run_story_evaluation(cmdline_args.stories,
-                         _agent,
-                         cmdline_args.max_stories,
-                         cmdline_args.failed,
-                         cmdline_args.output,
-                         cmdline_args.fail_on_prediction_errors,
-                         cmdline_args.e2e)
+    if cmdline_args.output:
+        nlu_utils.create_dir(cmdline_args.output)
+
+    if cmdline_args.mode == 'default':
```

Comment:
given that we do two different things, I think it makes sense to use subparsers to better group the arguments https://pymotw.com/2/argparse/#nesting-parsers